pages/common/git-extras.md

```diff
@@ -0,0 +1,16 @@
+# git extras
+
+> Set of commands for Git.
```

Comment:
`Unofficial Git extension pack.` would also be nice. 😄

Reply:
Yeah, I the current description is way too bland.  "Unofficial" though is too negative, I think 😝

How about that minus "unofficial"?
```suggestion
> Git extension pack.
```